javascript/packages/cli/src/cli.ts

```diff
@@ -32,6 +33,8 @@ import {
   DEFAULT_PROVIDER,
 } from "./constants";
 
+// We always want to log when the orchestrator is called from the cli
+const SILENT = false;
```

Comment:
No reason for the CLI to even know about the "silent" or for this to exist 
```suggestion
```

Reply:
It seems reasonable, let me refactor this. Thanks!